modin/pandas/series.py

```diff
@@ -1301,7 +1301,18 @@ def mul(self, other, level=None, fill_value=None, axis=0):  # noqa: PR01, RT01,
             new_other, level=level, fill_value=None, axis=axis
         )
 
-    multiply = rmul = mul
+    multiply = mul
+
+    def rmul(
+        self, other, level=None, fill_value=None, axis=0
+    ):  # noqa: PR01, RT01, D200
+        """
+        Return multiplication of series and `other`, element-wise (binary operator `mul`).
```

Comment:
```suggestion
        Return multiplication of `other` and series, element-wise (binary operator `mul`).
```

Reply:
I agree that what you suggest is clearer, but I would like to keep this as is to match the pandas API documentation.